proof_iavl_absence.go

```diff
@@ -53,16 +53,16 @@ func AbsenceOpDecoder(pop tmmerkle.ProofOp) (merkle.ProofOperator, error) {
 	if err != nil {
 		return nil, err
 	}
-	proof, err := rangeProofFromProto(pbProofOp.Proof)
+	proof, err := RangeProofFromProto(pbProofOp.Proof)
```

Comment:
Maybe these should be moved into an internal package or the `proto` package or something, don't think we want them to become part of the public API.

Reply:
what about the `utils`? module?

Reply:
That will wind up in the base package (all files in a directory make up a package). I'd rather not have this litter the base package public API, see e.g. the `Blue` type in the API reference for why I think this is bad:

https://pkg.go.dev/github.com/tendermint/iavl?tab=doc#pkg-index

Reply:
Good point, I guess we can make these part of the public API then - i.e. leave them as is.